### PR TITLE
FIX: Revise tear-down protocol and remove outdated steps

### LINKED ISSUE #166: Confirm shutdown procedure of the ET computer
As Celine mentioned, https://github.com/TheAxonLab/hcph-sops/pull/152#discussion_r1310575993, it is unclear that the following is true:

https://github.com/TheAxonLab/hcph-sops/blob/7da38633627eb2d0bc82f56d0cac1cdc966785dd/docs/data-collection/tear-down.md#L82

### PR DESCRIPTION
**Pull request by oesteban**
These edits were prompted by the revision needed about how to switch the ET computer off.

Resolves: #166.

### CODE REVIEW

File: docs/data-collection/tear-down.md [hunk 1]
```diff
@@ -32,30 +33,21 @@
...
-- [ ] Put the gloves on and detach the infrared mirror from the standard mirror.
+- [ ] Put the gloves on and cover the infrared mirror for storage.
 
     !!! danger "The infrared mirror MUST be manipulated with clean gloves at all times."
 
-- [ ] Immediately insert the infrared mirror back into its plastic pouch.
-- [ ] Take the pouch containing the infrared mirror outside to the Control Room and place it back in its designated box, with extreme care.
-- [ ] Clean the standard mirror removing all residues of glue from the scotch tape.
-- [ ] Re-attach the mirror to the coil's frame.
+- [ ] Re-attach the standard mirror to the coil's frame.
```
Comment (esavary) [inline docs/data-collection/tear-down.md]: Usually, there is no standard mirror on the coil, so we can probably remove this step.

**Review by esavary — APPROVED** [review]

File: docs/data-collection/notes-scanning.md [hunk 2]
```diff
@@ -127,6 +127,17 @@
...
 
+### Scanner shutdown protocol
+
+!!! warning "Do not switch the scanner off if some data are yet to be archived"
+
+![on-off-button](../assets/images/on-off-box.jpg)
+
+- [ ] Turn off the satellite station ({{ secrets.hosts.console_left | default("███") }}, the computer on the left side of the control desk)
+- [ ] Turn off the control station ({{ secrets.hosts.console_right | default("███") }}, the computer on the right side of the control desk)
+- [ ] Push the blue button displaying an overdotted circle and the **SYSTEM OFF** label above, which is found right above the key
```
Comment (acionca) [inline docs/data-collection/notes-scanning.md]: I would add a warning saying that you should wait for the control station to be off before shutting down the system (thus any remaining killing mouse pointer).
```suggestion
- [ ] Turn off the control station ({{ secrets.hosts.console_right | default("███") }}, the computer on the right side of the control desk)
- [ ] Wait for both computers to shut down.
- [ ] Push the blue button displaying an overdotted circle and the **SYSTEM OFF** label above, which is found right above the key
```

**Review by acionca — APPROVED** [review]
Looking good

File: docs/data-collection/tear-down.md (hunk 1 above)
Comment (oesteban) [reply docs/data-collection/tear-down.md]:
```suggestion
```

**Pull request merged**